addons/fastroping/XEH_postInit.sqf

```diff
@@ -32,6 +32,17 @@
 }, {false}] call CBA_fnc_addKeybind;
 
 
+if (isServer) then {
+    ["Helicopter", "init", {
+        if (!GVAR(autoAddFRIES)) exitWith {};
+        params ["_vehicle"];
+        if (!isNumber (configOf _vehicle >> QGVAR(enabled)) && {isNil {_vehicle getVariable [QGVAR(FRIES), nil]}}) then {
```

Comment:
Shouldn't this check if FRIES are supported on the vehicle?
ref: https://github.com/acemod/ACE3/blob/fd48ba87d77cf84be8958cf81e5b9d6986372ba6/addons/fastroping/functions/fnc_equipFRIES.sqf#L21

```suggestion
        if (isNumber (configOf _vehicle >> QGVAR(enabled)) && {isNil {_vehicle getVariable [QGVAR(FRIES), nil]}}) then {
```

Reply:
Ah yes, added that after as protection and just copied the `exitWith`. 🤦